qxf2_scheduler/routes.py

```diff
@@ -65,73 +68,89 @@ def index():
 
 
 @app.route("/interviewers")
-def listinterviewer():
-    "List the interviewer names,designation"
-    interviewer_work_time_slots = []
-    new_slot = Interviewers.query.join(Interviewertimeslots,Interviewers.interviewer_id==Interviewertimeslots.interviewer_id).values(Interviewers.interviewer_id,Interviewers.interviewer_email,Interviewertimeslots.interviewer_start_time,Interviewertimeslots.interviewer_end_time)
+def listinterviewers():
```

Comment:
Can you please change the method name to be list_interviewers()

Reply:
Changed the method name